### PR TITLE
refactor(content): added prev invisible content

### PR DESCRIPTION
**Pull request by aidana12**
@InstantGabe could you take a look at the content? Some sections in development have more content options than in the design (e.g. in the design there might not be a visible pretitle, but most sections have it), so we have some rogue content that needs to be checked. In the first commit you can see I added "new" preview content, would be good if you go over it. I've attached a screenshot of the changes I made

### CODE REVIEW

File: src/blocks/Collage/index.tsx [hunk 1]
```diff
@@ -260,6 +260,7 @@ export default defineBlock({
     fields: {
       pretitle: {
         type: "text",
+        preview: "New in",
```
Comment (aidana12) [inline src/blocks/Collage/index.tsx]: Screenshot 2023-03-13 at 18 00 07

File: src/blocks/ContentCard/index.tsx [hunk 2]
```diff
@@ -250,6 +250,7 @@ export default defineBlock({
     fields: {
       pretitle: {
         type: "text",
+        preview: "New in",
```
Comment (aidana12) [inline src/blocks/ContentCard/index.tsx]: Screenshot 2023-03-13 at 18 00 15

File: src/blocks/ProductSlider/index.tsx [hunk 3]
```diff
@@ -374,6 +374,7 @@ export default defineBlock({
       pretitle: {
         type: "text",
         label: "Pretitle",
+        preview: "Bestsellers",
```
Comment (aidana12) [inline src/blocks/ProductSlider/index.tsx:377]: Screenshot 2023-03-13 at 18 00 53

The products will be taken from the corresponding storefront, this we cannot really change, the texts could be better though

File: src/blocks/SocialProof/index.tsx [hunk 4]
```diff
@@ -171,19 +171,19 @@ export default defineBlock({
       pretitle: {
         type: "text",
         label: "Pretitle",
-        preview: "",
+        preview: "From A to Z",
```
Comment (aidana12) [inline src/blocks/SocialProof/index.tsx:174]: Screenshot 2023-03-13 at 18 01 01

**Comment by aidana12** [comment]
Other sections' content, would be good to confirm
Hero:
Screenshot 2023-03-13 at 18 00 39

Announcement banner
Screenshot 2023-03-13 at 18 00 00

Product CTA
Screenshot 2023-03-13 at 17 59 53

File: src/blocks/ContentCard/index.tsx (hunk 2 above)
Comment (InstantGabe) [inline src/blocks/ContentCard/index.tsx]: I would go for "Just in"

File: src/blocks/Collage/index.tsx (hunk 1 above)
Comment (InstantGabe) [inline src/blocks/Collage/index.tsx]: I would go for "Just in"

**Review by InstantGabe — CHANGES_REQUESTED** [review]
Looks good overall!

**Pull request merged**